distribution/docker/ddf/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine as prep
+
+RUN apk add --no-cache unzip
+RUN mkdir /prep /zips
+COPY maven/ddf-${project.version}.zip /zips
+RUN unzip /zips/ddf-${project.version}.zip -d /prep
+RUN mv /prep/ddf-${project.version} /prep/ddf
+
+FROM codice/ddf-base:${docker.ddf.base.version}
+LABEL maintainer=codice
+
+ENV APP_HOME=/app
+ENV APP_NAME=/ddf
+ENV APP_LOG=$APP_HOME/data/log/$APP_NAME.log
+
+COPY --from=prep /prep/ddf $APP_HOME
+
+VOLUME $APP_HOME/data $APP_HOME/deploy $APP_HOME/etc
+
+EXPOSE 8993 8181 8101
```

Comment:
❓Should we expose port 5005 and provide an option to turn on remote debugging when the container is started?

Reply:
I'm going to leave this out since the `EXPOSE` instruction is largely for communicating to anyone running the container what ports are intended to be published. Currently passing in an `env` var of `KARAF_DEBUG=true` will cause port `5005` to be bound inside the container, the actual exposing of that port is something left up to the person running the container regardless of what is in the `EXPOSE` instruction in the `Dockerfile`

Reply:
❓ There are two docker files. Why? Is one to run DDF itself in a container?